src/helpers/socketIo.js

```diff
@@ -0,0 +1,23 @@
+import socketIO from 'socket.io';
+import eventEmitters from './eventEmitters';
+
+const socketIo = (server) => {
+  const io = socketIO(server);
+  io.use((sockets, next) => {
+    if (sockets) {
+      sockets.socket = sockets;
+      next();
+    } else {
+      next(new Error('Socket Error'));
+    }
+  });
+
+  io.on('connection', (socket) => {
+    eventEmitters.on('notification_message', (data) => {
+      socket.emit('approve_reject', data);
+    });
+  });
+  return io;
+};
```

Comment:
Nice work @NiyoEric , I feel like your work will serve as a foundation to the team as we enter the socket.io world